src/AddIns/Uno.UI.MediaPlayer.Skia.X11/X11MediaPlayerPresenterExtension.cs

```diff
@@ -12,12 +12,6 @@
 using Uno.UI.Runtime.Skia;
 using Uno.WinUI.Runtime.Skia.X11;
 
-[assembly: ApiExtension(
-	typeof(IMediaPlayerPresenterExtension),
-	typeof(Uno.UI.MediaPlayer.Skia.X11.X11MediaPlayerPresenterExtension),
-	ownerType: typeof(MediaPlayerPresenter),
-	operatingSystemCondition: "linux")]
-
```

Comment:
Why remove those? It was working properly as it was avoiding the registration on non-linux systems.

Reply:
Ah I see, you're not registering it when not running. I don't think this is appropriate, as we'll load libvlc even if we don't need it. We'll need to find another way.